drivers/usb/device/usb_dc_kinetis.c

```diff
@@ -474,13 +471,13 @@ int usb_dc_ep_is_stalled(const u8_t ep, u8_t *const stalled)
 		*stalled = dev_data.ep_ctrl[ep_idx].status.in_stalled;
 	}
 
-#if defined(CONFIG_SYS_LOG) && (SYS_LOG_LEVEL >= SYS_LOG_LEVEL_WARNING)
+#if defined(CONFIG_LOG) && (LOG_LEVEL >= LOG_LEVEL_WARNING)
```

Comment:
redundant #if. When log level < warning compiler will figure out to not compile bd_idx and will not generate unused variable warning.